drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/java/org/drools/workbench/screens/drltext/backend/server/indexing/IndexRuleAttributeNameAndValueCompositionTest.java

```diff
@@ -47,7 +47,7 @@ public void testIndexDrlRuleAttributeNameAndValueComposition() throws IOExceptio
 
         Thread.sleep( 5000 ); //wait for events to be consumed from jgit -> (notify changes -> watcher -> index) -> lucene index
 
-        List<String> index = Arrays.asList(KObjectUtil.toKCluster(basePath.getFileSystem()).getClusterId());
+        List<String> index = Arrays.asList(KObjectUtil.toKCluster(basePath).getClusterId());
```

Comment:
Optional nitpick: In case you're creating just one element list `Collections.singletonList(..)` is [preferable](https://stackoverflow.com/questions/26027396/arrays-aslist-vs-collections-singletonlist) to `Arrays.asList(..)`.